gitpod-io/gitpod
# Use a normal component and conditionally build it using leeway build

## Description


See https://github.com/gitpod-io/gitpod/issues/13313 for context for why we're moving away from a dynamic package. This PR ensures that we can build `install/preview:docker` in Werft again. There's a small overhead in running "leeway build" twice, but it does not appear to be bad and it is only relevant when 'with-local-preview' is enabled.

## Related Issue(s)


https://github.com/gitpod-io/gitpod/issues/13313

## How to test


See [the job](https://werft.gitpod-dev.com/job/gitpod-build-mads-hartmann-build-enable-the-13313.3 )

<img width="1493" alt="Screenshot 2022-09-27 at 14 47 27" src="https://user-images.githubusercontent.com/83561/192530506-384fb3dd-e41e-4d12-9fa5-3e7da086d018.png">

## Release Notes

```release-note
NONE
```

## Documentation

N/A

## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
